ci/prow/update_all_boskos_projects.sh

```diff
@@ -38,6 +38,6 @@ if [[ -z "${BOSKOS_PROJECTS}" ]]; then
 fi
 
 for boskos_project in ${BOSKOS_PROJECTS}; do
-  # Set permissions for this project
-  "./set_boskos_permissions.sh" ${boskos_project} $@
+  # Set up this project
+  "./set_up_boskos_project.sh" ${boskos_project} $@
```

Comment:
```suggestion
  ./set_up_boskos_project.sh ${boskos_project} $@
```

Reply:
Done.
It's a bit strange that I cannot apply the changes here through the UI..